js/shared/input.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2014-2015 runtime.js project authors
```

Comment:
`service/shell` should probably use input provided by tty stdin interface instead of input-box directly